File: 944. Delete Columns to Make Sorted/main.py
class Solution:
    def minDeletionSize(self, strs: List[str]) -> int:
        cnt = 0
        for i in range(len(strs[0])):
            for j in range(1, len(strs)):
                print(strs[j-1][i], strs[j][i])
                if strs[j-1][i] > strs[j][i]:
                    cnt += 1
                    break

        return cnt

File: 944. Delete Columns to Make Sorted/main.js
/**
 * @param {string[]} strs
 * @return {number}
 */
var minDeletionSize = function (strs) {
    let cnt = 0;

    for (let i = 0; i < strs[0].length; i++) {
        for (let j = 1; j < strs.length; j++) {
            if (strs[j - 1][i] > strs[j][i]) {
                cnt++;
                break;
            }
        }
    }

    return cnt;
};